Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/TriangularFEMForceField.inl

```diff
@@ -1157,10 +1157,41 @@ void TriangularFEMForceField<DataTypes>::addForce(const core::MechanicalParams*
     if (f_computePrincipalStress.getValue() || p_computeDrawInfo)
     {
         unsigned int nbTriangles = m_topology->getNbTriangles();
-        type::vector<TriangleInformation>& triangleInf = *(triangleInfo.beginEdit());
+        auto triangleInf = sofa::helper::getWriteOnlyAccessor(triangleInfo);
         for (unsigned int i = 0; i < nbTriangles; ++i)
             computePrincipalStress(i, triangleInf[i].stress);
-        triangleInfo.endEdit();
+
+
+        if (showStressValue.getValue()) // if true will compute averageStress per point
+        {
+            auto triangles = m_topology->getTriangles();
+            auto vertexInf = sofa::helper::getWriteOnlyAccessor(vertexInfo);
+
+            m_minStress = std::numeric_limits<Real>::max();
+            m_maxStress = std::numeric_limits<Real>::min();
```

Comment:
```suggestion
            m_maxStress = std::numeric_limits<Real>::lowest();
```
min() will return the minimum positive value (0 more or less), lowest the biggest negative value.